src/popup/index.html

```diff
@@ -307,6 +307,12 @@ <h2><img src="/icon/icon.png" alt="🅱️" style="height: 1.2em" />lue Blocker<
 					<span name="popup-timer-value"></span>
 				</div>
 			</div>
+			<div class="blockstrings">
+				<div class="option">
+					<p>Block usernames with these words:</p>
```

Comment:
This text is a little unclear about what the setting does.
```suggestion
					<p>Block users with these words in their display name or username:</p>
```